python/paddle/fluid/tests/unittests/mkldnn/test_activation_bf16_mkldnn_op.py

```diff
@@ -104,3 +104,21 @@ def set_attrs(self):
 class TestMKLDNNGeluTanhDim2BF16Op(TestMKLDNNGeluTanhBF16Op):
     def init_data(self):
         self.x = np.random.uniform(-1, 1, [11, 17]).astype(np.float32)
+
+
+class TestMKLDNNReluBF16Op(TestMKLDNNSigmoidBF16Op):
+    def config(self):
+        self.op_type = "relu"
+
+    def init_dtype(self):
+        self.dtype = np.uint16
```

Comment:
You don't need that function since setting dtype is done in setUp function of TestMKLDNNSigmoidBF16Op class
https://github.com/PaddlePaddle/Paddle/blob/6629f1f77e9174e410e90c887d7a7595ace974aa/python/paddle/fluid/tests/unittests/mkldnn/test_activation_bf16_mkldnn_op.py#L44